homework/final/StudentsMarks/StudentsMarks/Data/StudentMark.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace StudentsMarks;
+
+public class StudentMark
+{
+    public int StudentMarkId { get; set; }
+    public string Name { get; set; } = "";
+    public string Course { get; set; } = "";
```

Comment:
```suggestion
    [Required(ErrorMessage = "Please student's name")]
    public string Name { get; set; } = "";
    [Required(ErrorMessage = "Please student's name")]
    public string Course { get; set; } = "";
```
Нет валидации полей
(-1)